CHANGELOG.md

```diff
@@ -8,6 +8,10 @@ and this project adheres to
 
 ## [Unreleased]
 
+### Fixed
+
+- Template missing necessary publish build.
```

Comment:
Minor, this doesn't really read like an action performed, it reads more like a statement or fact.